lib/corpus.py

```diff
@@ -383,13 +383,26 @@ def __init__(
         self.start = start
         self.end = end
         self.track = track
-        self.orth = orth
+        self._orth = orth
```

Comment:
I con'g understand this change?

Reply:
Typo from a previous version in which I used a property. Sorry about that, will revert back.